modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -1280,9 +1377,12 @@ func (suite *KeeperTestSuite) TestQueryUnreceivedAcks() {
 		{
 			"invalid seq",
 			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				suite.coordinator.Setup(path)
+
 				req = &types.QueryUnreceivedAcksRequest{
-					PortId:             "test-port-id",
-					ChannelId:          "test-channel-id",
+					PortId:             path.EndpointA.ChannelConfig.PortID,
+					ChannelId:          path.EndpointA.ChannelID,
```

Comment:
dito

Reply:
The original tests were wrong.

This test is the test that check [this if-statement ](https://github.com/Finschia/ibc-go/pull/15/commits/d1ae1c495142a1da95669243753a0562b518facc#file-modules-core-04-channel-keeper-grpc_query-go-L413)but the original code didn't cover it.